tools/dev-server.js

```diff
@@ -13,34 +13,69 @@ const { Builder } = require("./builder");
 const KARMA_PORT = 9876;
 const SERVE_PORT = 5000;
 
+class KarmaServer {
+  /**
+   * @param {string} configFile
+   * @param {string} [browser]
+   * @param {string} [grep]
+   */
+  constructor(configFile, browser, grep) {
+    this._configFile = configFile;
+    this._browsers = browser ? [browser] : undefined;
```

Comment:
Unless it conflicts with Karma, it might be better to use an empty array here instead of `undefined`.

Reply:
Realized late it was a mistake :sweat_smile: 
Undefined is important.